snap/snapcraft.yaml

```diff
@@ -198,12 +201,16 @@ parts:
       - bin/graphics-core22-wrapper
 
 plugs:
-  desktop-launch:
-  opengl:
   graphics-core22:
     interface: content
     target: $SNAP/graphics
     default-provider: mesa-core22
+  fonts:
+    interface: system-files
+    read:
+      - /etc/fonts
+      - /usr/share/fonts
+      - /usr/share/fonts/truetype
```

Comment:
I think we will also need "/usr/share/fonts/TTF" for Arch. (C.f. https://github.com/MirServer/mir/blob/19a573fcb5efd50f1c7e449c76973fb69cf106fe/src/server/shell/decoration/renderer.cpp#L90)

Reply:
Having slept on it… shouldn't we just use everything from inside the snap? And avoid the contentious `system-files` interface altogether?

Reply:
Yes, I would prefer that

Reply:
> Yes, I would prefer that

Done.